src/components/CloudCard/styles.scss

```diff
@@ -1,3 +1,17 @@
+html[data-theme="dark"] {
+  .cloud-card {
+    background-color: black;
+  }
+}
+
+html[data-theme="light"] {
+  --ifm-color-text: #383838;
+}
+
+html[data-theme="dark"] {
+  --ifm-color-text: #e3e3e3;
+}
```

Comment:
I'm concerned this might impact other pages or components. Maybe it's safe, I'm not sure how docusaurus scopes CSS in components. If you know this is safe, disregard this comment. Otherwise I recommend to scope these to `.cloud-card`.